static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -129,30 +129,6 @@ function SetupDocs({organization, projects, search}: Props) {
     <DocsWrapper key={platformDocs.html}>
       <Content dangerouslySetInnerHTML={{__html: platformDocs.html}} />
       {missingExampleWarning()}
-      {project && (
```

Comment:
@Neo-Zhixing why did this move?

Reply:
Otherwise the footer would disappear and show up again whenever you switch between projects. This needs to be outside your `DocsWrapper` which is a `motion.div`